### ISSUE TITLE
Dropdown contents not visible in high contrast (white) mode when dropdown is inside the details list

### ISSUE THREAD

**Issue by niadak** [OfficeDev/office-ui-fabric-react#4373]
Dropdown contents not visible in high contrast (white) mode when dropdown is inside the details list

![highcontrastissue](https://user-images.githubusercontent.com/11937515/37960153-7e753f16-31d2-11e8-8722-c0133d6992f1.png)
